nodes/nodes/nodes-nodes-adding-node-iso.adoc

```diff
@@ -24,6 +25,22 @@ The following platforms are supported for this method of adding nodes:
 * `none`
 * `external`
 
```

Comment:
In the section below, I interchangeably use wording like supported, validated, and tested. But if there's one word that I should consistently use throughout for clarity, let me know!

Reply:
seems ok to me